### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by FelipeEnne**
Wait for a code review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,80 @@
...
+<br>
+## Desktop size
+![Screenshot Image](assets/images/screenshot1.png)
+<br><br>
+## Tablet size
+![Screenshot Image](assets/images/screenshot2.png)
+<br><br>
+## Mobile size
+![Screenshot Image](assets/images/screenshot3.png)
+<br>
```
Comment (ebukaume) [inline README.md:47]: These images didn't render. You may want to investigate the cause.

**Review by ebukaume — CHANGES_REQUESTED** [review]
## Changes Required! :stop_sign: 

Hi @FelipeEnne  :wave: 

Nice work so far. You are doing well very. Just a few things to fix here.

**You are not setting the image src correctly here; consider appending `./` in front of the relative path**
review-u-booty-image

**The height of your navbar at the medium breakpoint is really out of proportion. Please fix it.**
review-u-booty-nav-height

Finally, the `SIGN IN` element at the navbar is supposed to be a link.

**Kindly submit another code review after fixing those issues.**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1685 @@
...
+              </article>
+            </div>
+
+          </div>
+        </section>
+
+
+        <section class="col-12">
+          <div class="title-generic">
+            <img src="assets/images/NlogoS.png" alt="description">
```
Comment (Redvanisation) [inline index.html]: The image is not showing please correct the image name `NLogoS.png` instead of `NlogoS .png`

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  🛑

### Hello @FelipeEnne   🌟

**Great work on this project but we still need to meet the following requirements first ⬇️** 

1. Please fix the image not showing, check my comment below about that.

2. Please Validate and fix the HTML markup [here](https://validator.w3.org/#validate_by_input)

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Review by Redvanisation — APPROVED** [review]
# Project approved ✔️ 

### Hello @FelipeEnne  🌟 

### _Nice work on this project, it seems that you have met the requirements so I am approving your project. 👍_ 

### You may merge to master and submit a project completion form. 🚀 

### Good luck! 🍀

File: index.html (hunk 2 above)
Comment (FelipeEnne) [reply index.html]: Ok, thanks

File: README.md (hunk 1 above)
Comment (FelipeEnne) [reply README.md:47]: Ok, thanks

**Pull request merged**